src/App.js

```diff
@@ -31,6 +60,22 @@ export default function App() {
   function handleClickBack() {
     setActivePage('LocationPage')
   }
+
+  function saveAsFavorite() {
+    setActivePage('FavoritePage')
+  }
+
+  function toFavorite() {
+    setActivePage('FavoritePage')
+  }
+
+  function handleBookmark(id) {
+    if (bookmarkedIds.some(favId => favId === id)) {
```

Comment:
```suggestion
    if (bookmarkedIds.some(bookmarkedId => bookmarkedId === id)) {
```
just to make it easier to read